README.md

```diff
@@ -42,3 +42,23 @@ ruby 2.6.8p205 (2021-07-07 revision 67951) [universal.arm64e-darwin21]
 - Run `flutter doctor` to ensure that everything works as expected from the command line.
 ### Running the app
 
+
+### Running the app wireless
```

Comment:
This section is not necessary with Android 11+.  Android studio supports Wireless Debugging. Maybe we include that above this and keep what you currently have if someone doesn't want to use wireless debugging... thoughts?

Reply:
Maybe you could show me how to do that hehe, I was unable to work it.